### DISCUSSION TITLE
用文本生成 Lo-Fi | AudioCraft & MusicGen 实战

### DISCUSSION THREAD

**Discussion by giscus** [iamStephenFang/stephenfang.github.io#9] [Blog Post Comments]
# 用文本生成 Lo-Fi | AudioCraft & MusicGen 实战

在网上冲浪的时候看到了 lofi generator 这个网站，试听了半小时音乐感觉质量都还可以，几乎与其他音乐平台听到的没差。从这个网站的 Licensing 页面了解到网站上所有音乐都是由AI生成的，并且网站还提供了付费的音乐下载选项。作为一个同时喜欢听Lo-Fi又喜欢鼓捣新鲜事物的人，有这样一个网站让我很是兴奋，但是我也很好奇这个网站是怎么做到的，于是我开始了我的探索之旅。

http://localhost:4321/posts/musicgenlofi

**Comment by GetToSet** [comment]
👋
